src/test/shell/bazel/bazel_toolchain_test.sh

```diff
@@ -22,8 +22,8 @@ CURRENT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 source "${CURRENT_DIR}/../integration_test_setup.sh" \
   || { echo "integration_test_setup.sh not found!" >&2; exit 1; }
 
-if [ "${PLATFORM-}" = "darwin" ]; then
-  echo "Skipping test: linaro toolchain is not supported on darwin host."
+if [ "${PLATFORM-}" = "darwin" -o "${MACHINE_TYPE}" = "ppc64le" ]; then
+  echo "Skipping test: linaro toolchain is not supported on darwin host or ppc64le Linux."
```

Comment:
After rebase, the test case started giving timeout error, even with checking linux instead of ppc64le or without any change i.e. just checking darwin. Is this a known problem? If it is, I'll undo this change and keep that original change of darwin.